test/test_opet_coin.js

```diff
@@ -1,145 +1,87 @@
-
-        try {
-            await token.sendAirdrops(airdropAccount, [1]);
-            assert.ifError('Error, too much tokens successfuly airdropped');
-        } catch (err) {
-            assert.equal(err, 'Error: VM Exception while processing transaction: revert', "Wrong error too big amount of airdrop");
-        }
+        let afterAirdropBalance0 = BigNumber(await token.balanceOf.call(airdropAccount[0])).toString();
+        let afterAirdropBalance1 = BigNumber(await token.balanceOf.call(airdropAccount[1])).toString();
+        assert.equal(afterAirdropBalance0, airdropAmount[0], 'Wrong address0 balance after token airdrop');
+        assert.equal(afterAirdropBalance1, airdropAmount[1], 'Wrong address1 balance after token airdrop');
     });
 });
 
 
 contract("Opet Token", async function (accounts) {
-    it("test transfer unable to execute", async function () {
+    it("test transfer paused is paused by default and unpause function works", async function () {
         let token = await Token.deployed();
-        let mintDestination = accounts[1];
+        let fromAccount = accounts[1];
         let transferDestination = accounts[2];
 
+        await token.transfer(fromAccount, 2);
 
-        await token.mint(mintDestination, 2);
         try {
-            await token.transfer(transferDestination, 1, {'from': mintDestination});
+            await token.transfer(transferDestination, 1, {'from': fromAccount});
             assert.ifError('Error, transfer should be unable');
         } catch (err) {
             assert.equal(err, 'Error: VM Exception while processing transaction: revert', "Transfer works before token unpaused");
         }
 
-        await token.approve(transferDestination, 1, {'from': mintDestination});
+        await token.approve(transferDestination, 1, {'from': fromAccount});
         try {
-            await token.transferFrom(mintDestination, transferDestination, 1, {'from': transferDestination});
+            await token.transferFrom(fromAccount, transferDestination, 1, {'from': transferDestination});
             assert.ifError('Error, transferFrom should be unable');
         } catch (err) {
             assert.equal(err, 'Error: VM Exception while processing transaction: revert', "transferFrom works before token unpaused");
         }
 
         await token.unpauseTransfer();
-        await token.transfer(transferDestination, 1, {'from': mintDestination});
+        await token.transfer(transferDestination, 1, {'from': fromAccount});
         let afterTransferBalance = BigNumber(await token.balanceOf(transferDestination));
         assert.equal(afterTransferBalance.toNumber(), 1, 'Wrong address balance after transfer');
 
-        await token.transferFrom(mintDestination, transferDestination, 1, {'from': transferDestination});
+        await token.transferFrom(fromAccount, transferDestination, 1, {'from': transferDestination});
```

Comment:
It's OK to keep it as it is, but in general, it would be better to have 2 test cases here - one to test the `transfer` and another to test the `transferFrom`.